Products/CMFDynamicViewFTI/tests/CMFDVFTITestCase.py

```diff
@@ -1,16 +1,18 @@
 # -*- coding: utf-8 -*-
-#
-# CMFDVFTITestCase
-#
 from plone.app import testing
-from plone.app.testing import bbb
+from plone.app.testing import PLONE_FIXTURE
+from plone.app.testing import PloneSandboxLayer
+from plone.app.testing.bbb import PloneTestCase
```

Comment:
That's a complete different topic, but it would be great if we could avoid using ``p.a.t.bbb`` module